### DISCUSSION TITLE
API down?

### DISCUSSION THREAD

**Discussion by jpmorris** [GSA-TTS/FAC#2182] [Q&A] [answered]
I had API requests working a few months ago, I'm trying again from the instructions here:

https://www.fac.gov/developers/getting-started/

And I get different errors

```
{
    "code": "PGRST000",
    "details": "could not translate host name \"cg-aws-broker-prodj2sek6m8sp1l07v.ci7nkegdizyy.us-gov-west-1.rds.amazonaws.com\" to address: Name does not resolve\n",
    "hint": null,
    "message": "Database connection error. Retrying the connection."
}
```

I assume this is a temporary outage.  However, I also got this earlier:

```
➜  ~ curl -s -X "GET" \
>      -H "X-Api-Key: ${API_GOV_KEY}" \
>      "${API_GOV_URL}/general?limit=5"
{"code":"42P01","details":null,"hint":null,"message":"relation \"api_v1_0_0.generallimit=5\" does not exist"}% 

```

**Comment by jadudm** [comment, marked as answer]
Our documentation at 

https://fac.gov/developers

has a bug. We had a trailing URL where we should not have. This caused this error.

Closing the loop here for completeness. We will patch the docs.

Many thanks!
